app/views/comparison/index.html.erb

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+<head>
+<style>
+#bills-table {
+  border-collapse: collapse;
+  width: 100%;
+}
+
+#bills-table td, #bills th {
+  border: 1px solid #ddd;
+  padding: 8px;
+}
+
+#bills-table tr:nth-child(even){background-color: #f2f2f2;}
+
+#bills-table tr:hover {background-color: #ddd;}
+
+#bills-table th {
+  padding-top: 12px;
+  padding-bottom: 12px;
+  text-align: left;
+  background-color: #4CAF50;
+  color: white;
+}
```

Comment:
This can probably go into the custom_css file that I have under assets

Reply:
Done! Thanks for the assistance.